### DISCUSSION TITLE
How can I contribuite to penpot as designer?

### DISCUSSION THREAD

**Discussion by callmehanu** [uxbox/uxbox#2467] [General]
Yep, I´m ui designer and really want to contribuite to penpot. 
Are there any way how designers can join to this project?

**Comment by elhombretecla** [comment]
Hi @callmehanu First of all, thank you very much for your interest ❤️

There is a [community post about design contribution](https://community.penpot.app/t/as-a-ux-ui-designer-what-can-i-do-to-help/826). There you can find more info.
